CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+<!-- 
+
+- Taking inspiration from https://github.com/zowe/docs-site/blob/master/README.md
+- Commenting out sections from that until we mature and need all their process
+
+-->
+<!-- uncomment once we get to the point of welcoming feedback 
+## Providing feedback
+
+Your feedback is essential in shaping the Zowe content experience. There are several ways to provide feedback:
```

Comment:
Your feedback is essential in shaping the Modernization Educational Repository.

Reply:
There are large portions of the Markdown in this file which are commented out for now... 

`<!-- uncomment once we get...` is just above.